test/system/libs/helpers.bash

```diff
@@ -577,3 +577,50 @@ function is_fedora_rawhide() (
 
   return 0
 )
+
+
+# Creates a container with org.freedesktop.Flatpak.SessionHelper D-Bus interface
+#
+# Pulling of an image is taken care of by the function
+#
+# Parameters:
+# ===========
+# - container_name - name of the container
+function create_container_flatpak_session_helper() (
+  local container_name="$1"
+
+  local default_image
+  default_image="$(get_default_image)"
+
+  pull_default_image
+
+  podman_cmd=(
+      podman create \
+          --dns none \
+          --env TOOLBOX_PATH="$TOOLBX" \
+          --name "$container_name" \
+          --network host \
+          --no-hosts \
+          --pid host \
+          --privileged \
+          --userns=keep-id \
+          --user root:root \
+          --volume "$HOME":"$HOME":rslave \
+          --volume "$TOOLBX":/usr/bin/toolbox \
+          --volume "$XDG_RUNTIME_DIR":"$XDG_RUNTIME_DIR" \
+          --volume "$XDG_RUNTIME_DIR/.flatpak-helper/monitor":/run/host/monitor \
+          --volume /etc:/run/host/etc \
+          --volume /usr:/run/host/usr:rslave \
+          "$default_image" \
+          toolbox init-container \
+                  --home "$HOME" \
+                  --shell "$SHELL" \
+                  --uid "$(id -ru)" \
+                  --user "$USER"
+    )
+
+    # Print the full command
+    # printf "%q " "${podman_cmd[@]}" >> /tmp/toolbox_podman_command.log
```

Comment:
We can use `bats --trace ...` for this.  It will show all the commands that get executed.